elephant/statistics.py

```diff
@@ -1360,10 +1361,13 @@ def _histogram_no_spread(self):
                                    self.bin_size,
                                    binary=self.binary)
 
+        time_hist_magnitude = time_hist.magnitude.flatten()
+        max_hist_entry = max(time_hist_magnitude)
+
         # Computing the histogram of the entries of pophist
         complexity_hist = np.histogram(
```

Comment:
**[optional] possible simplification ?**
Could this also be achieved by using np.bincount?, see: https://numpy.org/doc/stable/reference/generated/numpy.bincount.html


```suggestion
        complexity_hist = np.bincount(time_hist_magnitude)
```

If I understand correctly in the following example:
```python
time_histogram= np.array([0, 2, 0, 0, 1, 1, 0, 0, 1, 1,
                          0, 1, 1, 0, 0, 0, 2, 0, 1, 1])
```
we count the number of zeros, ones, and twos, so the desired result is:
```python
correct_complexity_histogram = np.array([10, 8, 2])
```

Reply:
After some experimenting, I think this simplification is in order, also for the border case of a zero-filled array